sources/rest_api/__init__.py

```diff
@@ -254,21 +257,39 @@ def paginate_resource(
                 client: RESTClient = client,
                 incremental_object: Optional[Incremental[Any]] = incremental_object,
                 incremental_param: IncrementalParam = incremental_param,
+                row_filter: Any = row_filter,
+                transform: Any = transform,
+                exclude_columns: List[str] = exclude_columns,
             ) -> Generator[Any, None, None]:
+                def exclude_elements(item: Any, exclude_columns: List[str]) -> Any:
+                    for key in exclude_columns:
+                        del item[key]
```

Comment:
I think a jsonpath would be significantly more powerful exclusion mechanism here

Reply:
used jsonpath